mcs_benchmark_data/_benchmark_submission_transformer.py

```diff
@@ -110,23 +118,61 @@ def __transform_submission(
 
             break
 
-    def __transform_submission_sample(
+    def _transform_commonsenseqa_submission_sample(
         self,
         submission_sample_jsonl_file_path: Path,
         submission_uri: URIRef,
-        submission_name: str,
     ) -> Generator[_Model, None, None]:
 
         with open(submission_sample_jsonl_file_path) as submission_sample_jsonl:
-            all_samples = list(submission_sample_jsonl)
 
-        for line in all_samples:
+            for line in submission_sample_jsonl:
 
-            sample = json.loads(line)
+                sample = json.loads(line)
 
-            yield SubmissionSample(
-                uri=URIRef(f"{submission_uri}:sample:{sample['id']}"),
-                submission_uri=submission_uri,
-                value=sample["chosenAnswer"],
-                about=f"{submission_name}-{sample['id']}",
-            )
+                yield SubmissionSample(

```

Comment:
This doesn't make sense as reusable code, either, because KagNet and RoBERTa have different information attached to submissions.